src/database/models/order.js

```diff
@@ -0,0 +1,39 @@
+const { v4: uuidv4 } = require('uuid');
```

Comment:
why you always use this package it doesn't help things better to use
this approach `Sequelize.UUID` in migration file while in modal `type: DataTypes.UUID`
and remember to give default value `defaultValue: DataTypes.UUIDV4` because we don't a lot of packages here this may slow our application